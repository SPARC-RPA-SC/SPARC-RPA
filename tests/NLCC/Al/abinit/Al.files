Al.in 
Al.out 
tbase_xi
tbase_xo 
tbase_x 
../psps/Al.psp8

