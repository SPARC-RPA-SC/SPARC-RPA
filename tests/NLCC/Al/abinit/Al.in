# Al in the diamond structure

# Definition of the k-point grid
ngkpt  1 1 1

shiftk   0 0 0
toldfe   1.0d-10
occopt 3
tsmear 0.01
nband 15
# Definition of the unit cell: fcc
acell 21.213203435596427  10.606601717798213  10.606601717798213 #Same parameters as Shiskin
rprim  0.0  0.5  0.5        #FCC primitive vectors (to be scaled by acell)
       0.5  0.0  0.5   
       0.5  0.5  0.0

# Definition of the atom types
ntypat  1         
znucl   13
                         
# Definition of the atoms
natom 2           # There are two atoms
typat  1 1     
xred              # Reduced coordinate of atoms
       0.0   0.01   0.02
       0.51  0.01   0.01

# Definition of the planewave basis set (at convergence 16 Rydberg 8 Hartree)
ecut 100.0          # Maximal kinetic energy cut-off, in Hartree


# Definition of the SCF procedure
# nstep   8        # For testing purpose only
diemac  12.0      # Although this is not mandatory, it is worth to
                  # precondition the SCF cycle. The model dielectric
                  # function used as the standard preconditioner
                  # is described in the "dielng" input variable section.
                  # Here, we follow the prescription for bulk silicon.

nsppol 2
spinat
0 0 1
0 0 1
