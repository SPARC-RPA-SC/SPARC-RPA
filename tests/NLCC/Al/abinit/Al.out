
.Version 8.10.3 of ABINIT 
.(MPI version, prepared for a x86_64_linux_gnu8.3 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 ABINIT comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Thu  3 Dec 2020.
- ( at 12h33 )
  
- input  file    -> Al.in
- output file    -> Al.out
- root for input  files -> tbase_xi
- root for output files -> tbase_xo


 Symmetries : space group C2/m (# 12); Bravais mC (1-face-center monocl.)
================================================================================
 Values of the parameters that define the memory need of the present run
     intxc =       0    ionmov =       0      iscf =       7    lmnmax =       6
     lnmax =       6     mgfft =     144  mpssoang =       3    mqgrid =    5514
     natom =       2  nloc_mem =       1    nspden =       2   nspinor =       1
    nsppol =       2      nsym =       4    n1xccc =    2501    ntypat =       1
    occopt =       3   xclevel =       2
-    mband =          15        mffmem =           1         mkmem =           1
       mpw =       14233          nfft =      746496          nkpt =           1
================================================================================
P This job should need less than                     407.373 Mbytes of memory.
  Rough estimation (10% accuracy) of disk space for files :
_ WF disk file :      6.517 Mbytes ; DEN or POT disk file :     11.393 Mbytes.
================================================================================

--------------------------------------------------------------------------------
------------- Echo of variables that govern the present computation ------------
--------------------------------------------------------------------------------
-
- outvars: echo of selected default values                                      
-   iomode0 =  0 , fftalg0 =312 , wfoptalg0 =  0
-
- outvars: echo of global parameters not present in the input file              
-  max_nthreads =    0
-
 -outvars: echo values of preprocessed input variables --------
            acell      2.1213203436E+01  1.0606601718E+01  1.0606601718E+01 Bohr
              amu      2.69815390E+01
           diemac      1.20000000E+01
             ecut      1.00000000E+02 Hartree
-          fftalg         312
           istwfk        2
              ixc          11
         kptrlatt        1    0    0      0    1    0      0    0    1
          kptrlen      7.50000000E+00
P           mkmem           1
            natom           2
            nband          15
            ngfft         144      72      72
             nkpt           1
           nspden           2
           nsppol           2
             nsym           4
           ntypat           1
              occ      1.000000  1.000000  1.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000
                       1.000000  1.000000  1.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000
           occopt           3
            rprim      0.0000000000E+00  5.0000000000E-01  5.0000000000E-01
                       5.0000000000E-01  0.0000000000E+00  5.0000000000E-01
                       5.0000000000E-01  5.0000000000E-01  0.0000000000E+00
          spgroup          12
           spinat      0.0000000000E+00  0.0000000000E+00  1.0000000000E+00
                       0.0000000000E+00  0.0000000000E+00  1.0000000000E+00
           symrel      1  0  0   0  1  0   0  0  1      -1  0  0   0 -1  0   0  0 -1
                       1  0 -2   0  1 -1   0  0 -1      -1  0  2   0 -1  1   0  0  1
            tnons      0.0000000  0.0000000  0.0000000    -0.4900000  0.0200000  0.0300000
                       0.0000000  0.0000000  0.0500000    -0.4900000  0.0200000 -0.0200000
           toldfe      1.00000000E-10 Hartree
            typat      1  1
           xangst      8.4191578345E-02  5.6127718897E-02  2.8063859448E-02
                       5.6127718897E-02  2.8905775232E+00  2.8905775232E+00
            xcart      1.5909902577E-01  1.0606601718E-01  5.3033008589E-02
                       1.0606601718E-01  5.4623998847E+00  5.4623998847E+00
             xred      2.2432820949E-19  1.0000000000E-02  2.0000000000E-02
                       5.1000000000E-01  1.0000000000E-02  1.0000000000E-02
            znucl       13.00000

================================================================================

 chkinp: Checking input parameters for consistency.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    2
   symrel is  -1  0  0  0 -1  0  0  0 -1
   tnons is    -4.900000E-01    2.000000E-02    3.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    2
   symrel is  -1  0  0  0 -1  0  0  0 -1
   tnons is    -4.900000E-01    2.000000E-02    3.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    2
   symrel is  -1  0  0  0 -1  0  0  0 -1
   tnons is    -4.900000E-01    2.000000E-02    3.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    3
   symrel is   1  0 -2  0  1 -1  0  0 -1
   tnons is     0.000000E+00    0.000000E+00    5.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    4
   symrel is  -1  0  2  0 -1  1  0  0  1
   tnons is    -4.900000E-01    2.000000E-02   -2.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    4
   symrel is  -1  0  2  0 -1  1  0  0  1
   tnons is    -4.900000E-01    2.000000E-02   -2.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

 chkinp: WARNING -
   Chksymbreak=1 . Found potentially symmetry-breaking value of tnons, 
   which is neither a rational fraction in 1/8th nor in 1/12th :
   for the symmetry number    4
   symrel is  -1  0  2  0 -1  1  0  0  1
   tnons is    -4.900000E-01    2.000000E-02   -2.000000E-02
   Please, read the description of the input variable chksymbreak,
   then, if you feel confident, you might switch it to zero, or consult with the forum.

================================================================================
== DATASET  1 ==================================================================
-   nproc =    1

 Exchange-correlation functional for the present dataset will be:
  GGA: Perdew-Burke-Ernzerhof functional - ixc=11
 Citation for XC functional:
  J.P.Perdew, K.Burke, M.Ernzerhof, PRL 77, 3865 (1996)

 Real(R)+Recip(G) space primitive vectors, cartesian coordinates (Bohr,Bohr^-1):
 R(1)=  0.0000000 10.6066017 10.6066017  G(1)= -0.0471405  0.0471405  0.0471405
 R(2)=  5.3033009  0.0000000  5.3033009  G(2)=  0.0942809 -0.0942809  0.0942809
 R(3)=  5.3033009  5.3033009  0.0000000  G(3)=  0.0942809  0.0942809 -0.0942809
 Unit cell volume ucvol=  5.9662135E+02 bohr^3
 Angles (23,13,12)=  6.00000000E+01  6.00000000E+01  6.00000000E+01 degrees

 getcut: wavevector=  0.0000  0.0000  0.0000  ngfft= 144  72  72
         ecut(hartree)=    100.000   => boxcut(ratio)=   2.13258

--- Pseudopotential description ------------------------------------------------
- pspini: atom type   1  psp file is ./../../../../psps_GGA/Al.psp8
- pspatm: opening atomic psp file    ./../../../../psps_GGA/Al.psp8
- Al    ONCVPSP-3.2.3.1  r_core=   1.76802   1.76802   1.70587
- 13.00000   3.00000    170504                znucl, zion, pspdat
    8   11    2    4       600   0.00000      pspcod,pspxc,lmax,lloc,mmax,r2well
    5.99000000000000    5.00000000000000    0.00000000000000   rchrg,fchrg,qchrg
     nproj     2     2     2
     extension_switch     1
  pspatm : epsatm=    4.08140610
         --- l  ekb(1:nproj) -->
             0    5.126667    0.728291
             1    7.287548    0.832437
             2   -2.770497   -0.637722
 pspatm: atomic psp has been read  and splines computed

   4.89768732E+01                                ecore*ucvol(ha*bohr**3)
--------------------------------------------------------------------------------

_setup2: Arith. and geom. avg. npw (full set) are   28465.000   28465.000

================================================================================

     iter   Etot(hartree)     deltaE(h) residm    vres2    magn
 ETOT  1  -4.7286514384597    -4.73E+00 1.93E-02 5.94E+02 0.000
 ETOT  2  -4.7362195905137    -7.57E-03 8.64E-04 5.13E+01 0.000
 ETOT  3  -4.7363663849763    -1.47E-04 3.75E-05 7.25E+00 0.000
 ETOT  4  -4.7363864516239    -2.01E-05 2.13E-05 5.00E-01 0.000
 ETOT  5  -4.7363878822788    -1.43E-06 1.64E-06 1.78E-02 0.000
 ETOT  6  -4.7363879586261    -7.63E-08 2.31E-07 1.29E-03 0.000
 ETOT  7  -4.7363879647330    -6.11E-09 2.42E-08 2.88E-04 0.000
 ETOT  8  -4.7363879658520    -1.12E-09 4.85E-09 4.67E-05 0.000
 ETOT  9  -4.7363879660442    -1.92E-10 4.90E-10 6.44E-06 0.000
 ETOT 10  -4.7363879660747    -3.04E-11 2.96E-10 2.45E-07 0.000
 ETOT 11  -4.7363879660758    -1.14E-12 6.04E-12 2.59E-09 0.000

 At SCF step   11, etot is converged : 
  for the second time, diff in etot=  1.143E-12 < toldfe=  1.000E-10

 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)=  6.73870919E-04  sigma(3 2)= -4.26007167E-04
  sigma(2 2)=  6.73870919E-04  sigma(3 1)=  4.26007167E-04
  sigma(3 3)=  6.73842778E-04  sigma(2 1)=  4.26076814E-04

 Integrated electronic and magnetization densities in atomic spheres:
 ---------------------------------------------------------------------
 Note: Diff(up-dn) is a rough approximation of local magnetic moment
 Atom    Radius    up_density   dn_density  Total(up+dn)  Diff(up-dn)
    1   2.00000     0.335701     0.335701      0.671401    -0.000000
    2   2.00000     0.335392     0.335392      0.670784    -0.000000
 ---------------------------------------------------------------------
  Sum:              0.671093     0.671093      1.342186    -0.000000
 Total magnetization (from the atomic spheres):            -0.000000
 Total magnetization (exact up - dn):                      -0.000000
================================================================================

 ----iterations are completed or convergence reached----

 Mean square residual over all n,k,spin=   29.877E-14; max=  60.430E-13
 reduced coordinates (array xred) for    2 atoms
       0.000000000000      0.010000000000      0.020000000000
       0.510000000000      0.010000000000      0.010000000000
 rms dE/dt=  4.2657E-02; max dE/dt=  7.3883E-02; dE/dt below (all hartree)
    1      -0.073883039775      0.000501734142     -0.000000000000
    2       0.073883039775     -0.000501734142      0.000000000000

 cartesian coordinates (angstrom) at end:
    1      0.08419157834476     0.05612771889650     0.02806385944825
    2      0.05612771889650     2.89057752316993     2.89057752316993

 cartesian forces (hartree/bohr) at end:
    1     -0.00353018384456     0.00353018384456     0.00343557594737
    2      0.00353018384456    -0.00353018384456    -0.00343557594737
 frms,max,avg= 3.4989321E-03 3.5301838E-03   0.000E+00  0.000E+00  0.000E+00 h/b

 cartesian forces (eV/Angstrom) at end:
    1     -0.18152933673533     0.18152933673533     0.17666440346726
    2      0.18152933673533    -0.18152933673533    -0.17666440346726
 frms,max,avg= 1.7992231E-01 1.8152934E-01   0.000E+00  0.000E+00  0.000E+00 e/A
 length scales= 21.213203435596 10.606601717798 10.606601717798 bohr
              = 11.225543779301  5.612771889650  5.612771889650 angstroms
 prteigrs : about to open file tbase_xo_EIG
 Fermi (or HOMO) energy (hartree) =   0.06671   Average Vxc (hartree)=  -0.25646
 Magnetization (Bohr magneton)=  3.25748317E-11
 Total spin up =  3.00000000E+00   Total spin down =  3.00000000E+00
 Eigenvalues (hartree) for nkpt=   1  k points, SPIN UP:
 kpt#   1, nband= 15, wtk=  1.00000, kpt=  0.0000  0.0000  0.0000 (reduced coord)
  -0.21069   -0.12386   -0.03559    0.17643    0.17659    0.20775    0.20793    0.21169
   0.27385    0.30606    0.30621    0.32604    0.32619    0.32712    0.35750
      occupation numbers for kpt#   1
   1.00000    1.00000    0.99996    0.00002    0.00002    0.00000    0.00000    0.00000
   0.00000    0.00000    0.00000    0.00000    0.00000    0.00000    0.00000
 Eigenvalues (hartree) for nkpt=   1  k points, SPIN DOWN:
 kpt#   1, nband= 15, wtk=  1.00000, kpt=  0.0000  0.0000  0.0000 (reduced coord)
  -0.21069   -0.12386   -0.03559    0.17643    0.17659    0.20775    0.20793    0.21169
   0.27385    0.30606    0.30621    0.32604    0.32619    0.32712    0.35750
      occupation numbers for kpt#   1
   1.00000    1.00000    0.99996    0.00002    0.00002    0.00000    0.00000    0.00000
   0.00000    0.00000    0.00000    0.00000    0.00000    0.00000    0.00000

--------------------------------------------------------------------------------
 Components of total free energy (in Hartree) :

    Kinetic energy  =  8.44323216258494E-01
    Hartree energy  =  8.07062086222520E-02
    XC energy       = -1.79331774227085E+00
    Ewald energy    = -3.88908454198048E+00
    PspCore energy  =  8.20903802329074E-02
    Loc. psp. energy= -7.55277364213490E-01
    NL   psp  energy=  6.94188753515424E-01
    >>>>> Internal E= -4.73637108983575E+00

    -kT*entropy     = -1.68762400550628E-05
    >>>>>>>>> Etotal= -4.73638796607580E+00

 Other information on the energy :
    Total energy(eV)= -1.28883671054773E+02 ; Band energy (Ha)=  -7.4026013056E-01
--------------------------------------------------------------------------------

 Cartesian components of stress tensor (hartree/bohr^3)
  sigma(1 1)=  6.73870919E-04  sigma(3 2)= -4.26007167E-04
  sigma(2 2)=  6.73870919E-04  sigma(3 1)=  4.26007167E-04
  sigma(3 3)=  6.73842778E-04  sigma(2 1)=  4.26076814E-04

-Cartesian components of stress tensor (GPa)         [Pressure= -1.9826E+01 GPa]
- sigma(1 1)=  1.98259636E+01  sigma(3 2)= -1.25335615E+01
- sigma(2 2)=  1.98259636E+01  sigma(3 1)=  1.25335615E+01
- sigma(3 3)=  1.98251357E+01  sigma(2 1)=  1.25356105E+01

== END DATASET(S) ==============================================================
================================================================================
  
 -outvars: echo values of variables after computation  --------
            acell      2.1213203436E+01  1.0606601718E+01  1.0606601718E+01 Bohr
              amu      2.69815390E+01
           diemac      1.20000000E+01
             ecut      1.00000000E+02 Hartree
           etotal     -4.7363879661E+00
            fcart     -3.5301838446E-03  3.5301838446E-03  3.4355759474E-03
                       3.5301838446E-03 -3.5301838446E-03 -3.4355759474E-03
-          fftalg         312
           istwfk        2
              ixc          11
         kptrlatt        1    0    0      0    1    0      0    0    1
          kptrlen      7.50000000E+00
P           mkmem           1
            natom           2
            nband          15
            ngfft         144      72      72
             nkpt           1
           nspden           2
           nsppol           2
             nsym           4
           ntypat           1
              occ      1.000000  1.000000  0.999964  0.000017  0.000017  0.000001
                       0.000001  0.000001  0.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000
                       1.000000  1.000000  0.999964  0.000017  0.000017  0.000001
                       0.000001  0.000001  0.000000  0.000000  0.000000  0.000000
                       0.000000  0.000000  0.000000
           occopt           3
            rprim      0.0000000000E+00  5.0000000000E-01  5.0000000000E-01
                       5.0000000000E-01  0.0000000000E+00  5.0000000000E-01
                       5.0000000000E-01  5.0000000000E-01  0.0000000000E+00
          spgroup          12
           spinat      0.0000000000E+00  0.0000000000E+00  1.0000000000E+00
                       0.0000000000E+00  0.0000000000E+00  1.0000000000E+00
           strten      6.7387091850E-04  6.7387091850E-04  6.7384277801E-04
                      -4.2600716679E-04  4.2600716679E-04  4.2607681355E-04
           symrel      1  0  0   0  1  0   0  0  1      -1  0  0   0 -1  0   0  0 -1
                       1  0 -2   0  1 -1   0  0 -1      -1  0  2   0 -1  1   0  0  1
            tnons      0.0000000  0.0000000  0.0000000    -0.4900000  0.0200000  0.0300000
                       0.0000000  0.0000000  0.0500000    -0.4900000  0.0200000 -0.0200000
           toldfe      1.00000000E-10 Hartree
            typat      1  1
           xangst      8.4191578345E-02  5.6127718897E-02  2.8063859448E-02
                       5.6127718897E-02  2.8905775232E+00  2.8905775232E+00
            xcart      1.5909902577E-01  1.0606601718E-01  5.3033008589E-02
                       1.0606601718E-01  5.4623998847E+00  5.4623998847E+00
             xred      2.2432820949E-19  1.0000000000E-02  2.0000000000E-02
                       5.1000000000E-01  1.0000000000E-02  1.0000000000E-02
            znucl       13.00000

================================================================================


- Timing analysis has been suppressed with timopt=0



================================================================================

 Suggested references for the acknowledgment of ABINIT usage.

 The users of ABINIT have little formal obligations with respect to the ABINIT group
 (those specified in the GNU General Public License, http://www.gnu.org/copyleft/gpl.txt).
 However, it is common practice in the scientific literature,
 to acknowledge the efforts of people that have made the research possible.
 In this spirit, please find below suggested citations of work written by ABINIT developers,
 corresponding to implementations inside of ABINIT that you have used in the present run.
 Note also that it will be of great value to readers of publications presenting these results,
 to read papers enabling them to understand the theoretical formalism and details
 of the ABINIT implementation.
 For information on why they are suggested, see also https://docs.abinit.org/theory/acknowledgments.
-
- [1] Recent developments in the ABINIT software package.
- Computer Phys. Comm. 205, 106 (2016).
- X.Gonze, F.Jollet, F.Abreu Araujo, D.Adams, B.Amadon, T.Applencourt,
- C.Audouze, J.-M.Beuken, J.Bieder, A.Bokhanchuk, E.Bousquet, F.Bruneval
- D.Caliste, M.Cote, F.Dahm, F.Da Pieve, M.Delaveau, M.Di Gennaro,
- B.Dorado, C.Espejo, G.Geneste, L.Genovese, A.Gerossier, M.Giantomassi,
- Y.Gillet, D.R.Hamann, L.He, G.Jomard, J.Laflamme Janssen, S.Le Roux,
- A.Levitt, A.Lherbier, F.Liu, I.Lukacevic, A.Martin, C.Martins,
- M.J.T.Oliveira, S.Ponce, Y.Pouillon, T.Rangel, G.-M.Rignanese,
- A.H.Romero, B.Rousseau, O.Rubel, A.A.Shukri, M.Stankovski, M.Torrent,
- M.J.Van Setten, B.Van Troeye, M.J.Verstraete, D.Waroquier, J.Wiktor,
- B.Xu, A.Zhou, J.W.Zwanziger.
- Comment : the fourth generic paper describing the ABINIT project.
- Note that a version of this paper, that is not formatted for Computer Phys. Comm. 
- is available at https://www.abinit.org/about/ABINIT16.pdf .
- The licence allows the authors to put it on the Web.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2016
-
- [2] Optimized norm-conserving Vanderbilt pseudopotentials.
- D.R. Hamann, Phys. Rev. B 88, 085117 (2013).
- Comment: Some pseudopotential generated using the ONCVPSP code were used.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#hamann2013
-
- [3] ABINIT : First-principles approach of materials and nanosystem properties.
- Computer Phys. Comm. 180, 2582-2615 (2009).
- X. Gonze, B. Amadon, P.-M. Anglade, J.-M. Beuken, F. Bottin, P. Boulanger, F. Bruneval,
- D. Caliste, R. Caracas, M. Cote, T. Deutsch, L. Genovese, Ph. Ghosez, M. Giantomassi
- S. Goedecker, D.R. Hamann, P. Hermet, F. Jollet, G. Jomard, S. Leroux, M. Mancini, S. Mazevet,
- M.J.T. Oliveira, G. Onida, Y. Pouillon, T. Rangel, G.-M. Rignanese, D. Sangalli, R. Shaltaf,
- M. Torrent, M.J. Verstraete, G. Zerah, J.W. Zwanziger
- Comment : the third generic paper describing the ABINIT project.
- Note that a version of this paper, that is not formatted for Computer Phys. Comm. 
- is available at https://www.abinit.org/about/ABINIT_CPC_v10.pdf .
- The licence allows the authors to put it on the Web.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2009
-
- [4] A brief introduction to the ABINIT software package.
- Z. Kristallogr. 220, 558-562 (2005).
- X. Gonze, G.-M. Rignanese, M. Verstraete, J.-M. Beuken, Y. Pouillon, R. Caracas, F. Jollet,
- M. Torrent, G. Zerah, M. Mikami, Ph. Ghosez, M. Veithen, J.-Y. Raty, V. Olevano, F. Bruneval,
- L. Reining, R. Godby, G. Onida, D.R. Hamann, and D.C. Allan.
- Comment : the second generic paper describing the ABINIT project. Note that this paper
- should be cited especially if you are using the GW part of ABINIT, as several authors
- of this part are not in the list of authors of the first or third paper.
- The .pdf of the latter paper is available at https://www.abinit.org/about/zfk_0505-06_558-562.pdf.
- Note that it should not redistributed (Copyright by Oldenburg Wissenshaftverlag,
- the licence allows the authors to put it on the Web).
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2005
-
- And optionally:
-
- [5] First-principles computation of material properties : the ABINIT software project. 
- X. Gonze, J.-M. Beuken, R. Caracas, F. Detraux, M. Fuchs, G.-M. Rignanese, L. Sindic,
- M. Verstraete, G. Zerah, F. Jollet, M. Torrent, A. Roy, M. Mikami, Ph. Ghosez, J.-Y. Raty, D.C. Allan.
- Computational Materials Science 25, 478-492 (2002). http://dx.doi.org/10.1016/S0927-0256(02)00325-7
- Comment : the original paper describing the ABINIT project.
- DOI and bibtex : see https://docs.abinit.org/theory/bibliography/#gonze2002
-
- Proc.   0 individual time (sec): cpu=         32.2  wall=         32.3

================================================================================

 Calculation completed.
.Delivered   7 WARNINGs and   2 COMMENTs to log file.
+Overall time at end (sec) : cpu=         32.2  wall=         32.3
